Wrappers/Python/cil/framework/framework.py

```diff
@@ -2136,11 +2135,11 @@ def dimension_labels(self):
         #remove from list labels where len == 1
         #
         for i, x in enumerate(shape_default):
-            if x == 0 or x==1:
-                try:
-                    labels.remove(labels_default[i])
-                except ValueError:
-                    pass #if not in custom list carry on
+                if x == 0 or x==1:
+                    try:
+                        labels.remove(labels_default[i])
+                    except ValueError:
+                        pass #if not in custom list carry on
```

Comment:
Remove added white space

```suggestion
            if x == 0 or x==1:
                try:
                    labels.remove(labels_default[i])
                except ValueError:
                    pass #if not in custom list carry on
```